cmd/release-controller/http.go

```diff
@@ -464,6 +465,42 @@ func (c *Controller) httpReleaseChangelog(w http.ResponseWriter, req *http.Reque
 	fmt.Fprintln(w, out)
 }
 
+func (c *Controller) httpReleaseInfoJson(w http.ResponseWriter, req *http.Request) {
+	start := time.Now()
+	defer func() { glog.V(4).Infof("rendered in %s", time.Now().Sub(start)) }()
+
+	tag := req.URL.Query().Get("tag")
+	if len(tag) == 0 {
+		http.Error(w, fmt.Sprintf("tag must be specified"), http.StatusBadRequest)
+		return
+	}
+
+	tags, ok := c.findReleaseStreamTags(false, tag)
+	if !ok {
```

Comment:
you asked for a single tag, you can just exit with the error if `!ok` and use the tag the user provided as input